Cargo.toml

```
@@ -1,16 +1,16 @@
 [package]
 name = "async_minreq"
-version = "2.13.3-alpha"
+version = "0.1.0"
 authors = ["Jens Pitkanen <jens@neon.moe>", "Beulah Evanjalain" ]
 description = "Simple, minimal-dependency HTTP client"
-# documentation = "https://docs.rs/minreq"
+# documentation = "https://docs.rs/async-minreq"
```

It should be fine to uncomment this since it will be valid once this initial version is published.

```suggestion
documentation = "https://docs.rs/async-minreq"
```